### ISSUE TITLE
Why is there no deb repository for this project?

### ISSUE THREAD

**Issue by paulkramme** [esell/deb-simple#23]
As this project is about a debian package server, is there a reason why there is none for this project? Could you create one?

**Comment by esell** [comment]
Certainly makes sense although since the app ends up being a single binary and a single config file it might be a bit overkill ;)

Maybe I'll use https://github.com/xor-gate/debpkg to build it!

**Comment by paulkramme** [comment]
Yeah, i think i will make my prohects available this way too... Although
some may require a more specialized way...

**Comment by esell** [comment]
Migrating to self-hosted git repo. Issue has been re-created here: https://git.esheavyindustries.com/esell/deb-simple/issues/5
